src/components/Party/Drawer/ActivityDrawerWidget.vue

```diff
@@ -0,0 +1,230 @@
+<template>
+    <div id="drawersContainer">
+        <div id="rightDrawer" class="drawer">
+            <input id="rightDrawerCheck" type="checkbox"/>
+            <div id="rightDrawerContents" class="drawerContents">
+                    <div v-for="action in actions" :key="action._id" class="drawer_item">
+                        <div class="stack">
+                            <div class="actionLine">
+                                <!-- TODO: Add origin player avatar here when added to API -->
+                                <!-- <img class="playerAvatar" :src="`/players/${player.icon}.png`" :alt="`${player.icon}.png`"> -->
+                                <!-- v-if="action.userId === user._id"    <label ><small>(You)</small></label> -->
+                                <div class="actionType"><strong>{{ actionSentence(action.actionType) }}</strong><br/>
+                                <label ><small>il y a {{ timeSince(action.date) }}</small></label></div>
+                            </div>
+                        </div>
+                    </div>
+            </div><label id="rightDrawerPull" class="drawerPull" for="rightDrawerCheck"></label>
+        </div>
+    </div>
+</template>
+
+<script>
+import {mapGetters} from 'vuex'
+
+export default {
+  name: "StatDrawerWidget",
+  props: {
+    actions: { type: Array },
+  },
+  methods: {
+    timeSince(date) {
```

Comment:
Next time you can use moment js, i love this library.

Reply:
Noted, good to know because I hesitated using it